tic-tac-toe/src/tic_tac_toe/board.clj

```diff
@@ -0,0 +1,79 @@
+(ns tic-tac-toe.board
+  (:require [tic-tac-toe.marks :refer :all]))
+
+(def dimension 3)
+
+(defn- one-based [value]
+  (+ value 1))
+
+(defn- three-matching-player-symbols? [row]
+  (let [[one two three] row]
+    (and (= one two three) (not(= one nil)))))
+
+(defn- winning-row? [rows]
+  (some true? (map
+                (fn[row] (three-matching-player-symbols? row))
```

Comment:
Hi @ukutaht - This one I couldn't get working - In this example, I am using the anonymous function to destruct:

`rows` into `[[row1] [row2] [row3]]` 

then the `three-matching-player-symbols` function further destructs 
`[row1]` into `[one two three]` so I can get each symbol in the row.

Is this still possible without using the anonymous function?